vendor/github.com/docker/libnetwork/drivers/overlay/joinleave.go

```diff
@@ -145,6 +145,21 @@ func (d *driver) Join(nid, eid string, sboxKey string, jinfo driverapi.JoinInfo,
 	return nil
 }
 
+func (d *driver) DecodeTableEntry(tablename string, key string, value []byte) (string, string) {
+	if tablename != ovPeerTable {
+		logrus.Errorf("DecodeTableEntry: unexpected table name %s", tablename)
+		return "", ""
+	}
+
+	var peer PeerRecord
+	if err := proto.Unmarshal(value, &peer); err != nil {
+		logrus.Errorf("DecodeTableEntry: failed to unmarshal peer record for key %s: %v", key, err)
+		return "", ""
+	}
+
+	return key, "Host IP: " + peer.TunnelEndpointIP
```

Comment:
Ok, I think its a good idea. If the driver wants to return more info in future a map would be better than cramming it all into one string. Will change it in the libnetwork PR.